service/src/main/java/uk/gov/hmcts/reform/fpl/service/DraftCMOService.java

```diff
@@ -90,8 +91,8 @@ public void prepareCustomDirections(CaseDetails caseDetails, CaseManagementOrder
         }
     }
 
-    public DynamicList getHearingDateDynamicList(List<Element<HearingBooking>> hearings, CaseManagementOrder order) {
-        List<DynamicListElement> values = getDateElements(hearings);
+    public DynamicList getHearingDateDynamicList(CaseData caseData, CaseManagementOrder order) {
```

Comment:
just a heads up, I have refactored this as a part of my PR for 1483, there is gonna be some conflicst 🙃

Reply:
my other comment is should this story not literally just be removing the filter in the old getDateElements method?

Reply:
@swalker125 when do you think you'll be merging your PR?

Reply:
@swalker125 just noticed your question. That would be the case, but we still need to filter out past hearings for action CMO (selecting next hearing date) + we want to exclude hearings that have an associated sealed CMO